README.md

```diff
@@ -10,7 +10,9 @@ An implementation of Resources in Ember.JS without decorators.
 _This is a [V2-format Addon](https://github.com/emberjs/rfcs/pull/507) with V1 compatibility_
```

Comment:
Somewhat related: Should the lines 6-10 be somewhere in a "notes" section down below. IMO most people won't care at that point of getting through the README.

Reply:
I've moved this to the bottom. Thanks!